src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1193,14 +1004,14 @@ void AppHost::_stopFrameTimer()
 //   is called as the `_frameTimer` Tick callback, roughly 60 times per second.
 void AppHost::_updateFrameColor(const winrt::Windows::Foundation::IInspectable&, const winrt::Windows::Foundation::IInspectable&)
 {
-    // - Convert the time delta between when we were started and now, to a hue. This will cycle us through all the colors.
-    // - Convert that hue to an RGB value.
-    // - Set the frame's color to that RGB color.
-    const auto now = std::chrono::high_resolution_clock::now();
-    const std::chrono::duration<float> delta{ now - _started };
-    const auto seconds = delta.count() / 4; // divide by four, to make the effect slower. Otherwise it flashes way to fast.
-    float ignored;
-    const auto color = til::color::from_hue(modf(seconds, &ignored));
+    LARGE_INTEGER freq, counter;
+    QueryPerformanceFrequency(&freq);
+    QueryPerformanceCounter(&counter);
+
+    const auto period = freq.QuadPart * 4;

```

Comment:
oh intriguing, all windows will have the same frame xolor now cool

Reply:
Oh, woops, I wanted to back out this change and submit it as a separate PR.
And yeah, that, and fixing floating imprecision once WT has run for a long time, were my intentions with this.